fe/fe-core/src/main/java/com/starrocks/planner/MysqlTableSink.java

```diff
@@ -69,4 +69,9 @@ public PlanNodeId getExchNodeId() {
     public DataPartition getOutputPartition() {
         return null;
     }
+
+    @Override
+    public boolean canUsePipeLine() {
+        return true;
+    }
```

Comment:
`DataSink#canTableSinkUsePipeline` should also be updated. Otherwise, it may still not use pipeline engine.

```diff
    public static boolean canTableSinkUsePipeline(Table table) {
        if (table instanceof OlapTable) {
            return Config.enable_pipeline_load;
+        } else if (table instanceof MysqlTable) {
+          return true;
        }
        return false;
    }
```

Reply:
> `DataSink#canTableSinkUsePipeline` should also be updated. Otherwise, it may still not use pipeline engine.
> 
> ```diff
>     public static boolean canTableSinkUsePipeline(Table table) {
>         if (table instanceof OlapTable) {
>             return Config.enable_pipeline_load;
> +        } else if (table instanceof MysqlTable) {
> +          return true;
>         }
>         return false;
>     }
> ```

fixed